frontend/integration-tests/views/operator-hub.view.ts

```diff
@@ -16,6 +16,13 @@ export const createSubscriptionFormTitle = element(by.cssContainingText('h1', 'C
 export const createSubscriptionFormBtn = element(by.buttonText('Subscribe'));
 export const createSubscriptionFormInstallMode = element(by.cssContainingText('label', 'Installation Mode'));
 
+export const installNamespaceDropdown = $('.dropdown--full-width');
```

Comment:
We might give this a more descriptive ID than we're currently using in the form, and then we can select the right dropdown by ID here.

Don't worry about this if the PR passes tests, though. We need to unblock the queue.